add banner image to site repo (#87)
carpentries-incubator/jekyll-pages-novice: _episodes/06-includes.md

But the principles behind reusing content apply regardless of
the particular framework being used to build the site.

## Carrying the Banner

Let's look at an example of how we can create a block of common content
and reuse it in multiple pages on our site.
At the moment our pages are quite plain:
why don't we try adding a banner to the top of each page?

We're going to add a pre-made banner image to our repository,
and it is good practice to store all image files in a common folder.
1. click "Create new file" under the "Add file"
dropdown on your repository homepage,
2. in the "Name your file..." box, type `images/`.
   The folder name should be automatically inserted in the path displayed
   next to this box for naming the file.
3. You can then leave the file blank and name it `.gitkeep`.
   When you commit the changes, the `images` folder will have been

>    added to your repository.

Reviewer: This seems like a good way to do this but we could also have them drag-drop the image and then rename it into the images folder.  I'm not sure which way would be easier.  Thoughts?

Reviewer: This does teach `.gitkeep` which could be good.

Author: Let's leave it for now and change if people find this too confusing in the first trial run next week

We will be uploading our banner image to this folder in a moment
and unfortunately GitHub does not provide a way to create a new folder
while uploading existing files, only while creating new ones.
When making these blank files,
which exist only to allow the existence of their parent repository,
it is traditional to call them `.gitkeep`.
Now [download this banner image that we will add to our pages][banner-image] and
upload the file to your newly-created `images` folder on GitHub:

> youc an do this by navigating into the folder and choosing

Reviewer: ```suggestion
you can do this by navigating into the folder and choosing
```

Author: addressed in 673ef5f

> > ## Solution
> >
> > ```

> > > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="/images/site_banner.png" alt="Group Website with Jekyll"></a>

Reviewer: 👍 for alt text

Reviewer: Maybe we should write why alt text is important?

Author: tried to address this in 4a2a87a
